integration_tests/dbt_project_evaluator/dbt_project.yml

```diff
@@ -0,0 +1,133 @@
+name: 'trino_utils_dbt_project_evaluator_integration_tests'
+version: '1.0.0'
+config-version: 2
+
+# This setting configures which "profile" dbt uses for this project.
+profile: 'integration_tests'
+
+# These configurations specify where dbt should look for different types of files.
+# The `source-paths` config, for example, states that models in this project can be
+# found in the "models/" directory. You probably won't need to change these!
+model-paths: ["models"]
+analysis-paths: ["analysis"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"  # directory which will store compiled SQL files
+clean-targets:         # directories to be removed by `dbt clean`
+  - "target"
+  - "dbt_packages"
+
+dispatch:
+  - macro_namespace: dbt
+    search_order: ['trino_utils_dbt_project_evaluator_integration_tests', 'trino_utils', 'dbt_project_evaluator', 'dbt']
+  - macro_namespace: dbt_project_evaluator
+    search_order: ['trino_utils_dbt_project_evaluator_integration_tests', 'trino_utils', 'dbt_project_evaluator']
+
+models:
+  trino_utils_dbt_project_evaluator_integration_tests:
+    # materialize as ephemeral to prevent the fake models from executing, but keep them enabled
+    +materialized: ephemeral
+  dbt_project_evaluator_integration_tests:
+    # materialize as ephemeral to prevent the fake models from executing, but keep them enabled
+    +materialized: ephemeral
+    marts:
+      intermediate:
+        to_exclude:
+          # Overridden by trino_utils_dbt_project_evaluator_integration_tests to_exclude
+          +enabled: false
+      fct_model_6:
+        +materialized: table
+  dbt_project_evaluator:
+    marts:
+      tests:
+        fct_test_coverage:
+          # materialize as a table to ensure SQL query runs successfully
+          +materialized: table
+      documentation:
+        fct_documentation_coverage:
+          # materialize as a table to ensure SQL query runs successfully
+          +materialized: table
+      structure:
+        # Overridden by trino__fct_model_naming_conventions
+        fct_model_naming_conventions:
+          +enabled: false
+      dag:
+        # Overridden by trino__fct_model_fanout
+        fct_model_fanout:
+          +enabled: false
+
+tests:
+  dbt_project_evaluator:
+    +enabled: false
+  dbt_project_evaluator_integration_tests:
+    dbt_project_evaluator_schema_tests:
+      unique_int_all_dag_relationships_path:
+        # Grouping by expressions of type ARRAY is not allowed for BigQuery
+        +enabled: "{{ false if target.type in ['bigquery'] else true }}"
```

Comment:
bigquery references

Reply:
removed

Reply:
It references [bigquery__recursive_dag](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L125), which is an implementation which uses loops with CTEs instead of recursive WITH ([default__recursive_dag implementation](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L5)).
As Trino recursive WITH queries have poor performance, we also want to use loops with CTEs in our implementation.
Although it is named "bigquery", it hasn't any bigquery specific logic and it works correctly with Trino. 
IMHO it is better to just use this macro, instead of copying and pasting entire logic, just to avoid reference to macro named bigquery.

[spark__recursive_dag](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L237-L240) is implemented in same way - it just reference bigquery__recursive_dag

WDYT?